test/lib/godmin/policy_finder_test.rb

```diff
@@ -1,48 +1,54 @@
 require "test_helper"
 
-module Namespace
-  class ArticlePolicyTestPolicy; end
-  class ObjectPolicy; end
-end
-
-class ArticlePolicyTest; extend ActiveModel::Naming; end
-class OverriddenPolicyTest
-  extend ActiveModel::Naming
-  def self.policy_class
-    Namespace::ObjectPolicy
-  end
-
-  def policy_class
-    Namespace::ObjectPolicy
-  end
-end
-
 module Godmin
   module Authorization
     class PolicyFinderTest < ActiveSupport::TestCase
+      class Article; end
+      class ArticlePolicy; end
```

Comment:
Yeah that was before when we put them in the global namespace. Now they're nested in the test module so everything's fine :+1: